mqr/midline_ussd.py

```diff
@@ -0,0 +1,689 @@
+        )
+
+    async def state_important_to_vaccinate(self):
+        question = self._("")
+        error = self._("Please use numbers from list.\n")
+        choices = [
+            Choice("strongly_agree", "I strongly agree"),
+            Choice("agree", "I agree"),
+            Choice("undecided", "I don't agree or disagree"),
+            Choice("disagree", "I disagree"),
+            Choice("strongly_disagree", "I strongly disagree"),
+            Choice("skip", "Skip"),
+        ]
+
+        return ChoiceState(
+            self,
+            question=question,
+            error=error,
+            choices=choices,
+            next="state_vaccine_benefits_outweighs_risk_question",
+        )
+
+    # split
+    async def state_vaccine_benefits_outweighs_risk_question(self):
+        question = self._(
+            "15/16\n"
+            "\n"
+            "What do you think about this statement?\n"
+            "\n"
+            "The benefits of vaccinating my child outweighs the risks my child will develop side effects from them"
+        )
+        error = self._("Please use numbers from list.\n")
+        choices = [Choice("1", "Next")]
+
+        return ChoiceState(
+            self,
+            question=question,
+            error=error,
+            choices=choices,
+            next="state_vaccine_benefits_outweighs_risk",
+        )
+
+    async def state_vaccine_benefits_outweighs_risk(self):
+        question = self._("")
+        error = self._("Please use numbers from list.\n")
+        choices = [
+            Choice("strongly_agree", "I strongly agree"),
+            Choice("agree", "I agree"),
+            Choice("undecided", "I don't agree or disagree"),
+            Choice("disagree", "I disagree"),
+            Choice("strongly_disagree", "I strongly disagree"),
+            Choice("skip", "Skip"),
+        ]
+
+        return ChoiceState(
+            self,
+            question=question,
+            error=error,
+            choices=choices,
+            next="state_likelihood_of_following_schedule",
+        )
+
+    async def state_likelihood_of_following_schedule(self):
+        question = self._(
+            "16/16\n"
+            "\n"
+            "How likely are you to follow the recommended shot schedule for your child?"
+        )
+        error = self._("Please use numbers from list.\n")
+        choices = [
+            Choice("very_unlikely", "Very unlikely"),
+            Choice("unlikely", "Unlikely"),
+            Choice("not_sure", "Not sure"),
+            Choice("likely", "Likely"),
+            Choice("very_likely", "Very likely"),
+            Choice("skip", "Skip"),
+        ]
+
+        return ChoiceState(
+            self,
+            question=question,
+            error=error,
+            choices=choices,
+            next="state_end",
+        )
+
+    # TODO FWB - Update all functions below this line (copied from baseline)
+    async def state_submit_data(self):
```

Comment:
Can be removed, the last question should go to state_update_rapidpro_contact

Reply:
Done.  Not sure if you meant just the "state_submit_data" function, but I also removed "state_already_completed" and "state_contact_not_found" as they weren't being called anywhere I could see